### DISCUSSION TITLE
How to migrate @vue/cli-service to Rspack

### DISCUSSION THREAD

**Discussion by cyesky** [web-infra-dev/rspack#7750] [Announcements]
### What subject does this issue tracking?

# [Subject]
@vue/cli-service如何迁移到Rspack
```
项目使用@vue/cli-service搭建，该场景下，webpack被内置在vue-cli-service脚本里。怎样才能让项目使用Rspack？卸载所有@vue/cli-service依赖，重新安装Rspack，并改造项目打包构建脚本命令吗
- [ ] Add a draft title or issue reference here
- [ ] 
```

**Comment by hardfist** [comment]
https://rsbuild.dev/guide/migration/vue-cli you can see vue-cli migration guide here

**Comment by djkloop** [comment]
直接rspack起个新项目，然后把代码一点点挪过去，我们项目当时迁移就是一点一点迁移的，除了sass问题其它几乎无缝迁移 我们是vue-cli 3

**Reply by mlkt** [reply]
差异这么大，根本不能叫兼容，就算80%相同，迁移要覆盖那20%不同的地方，几乎都可以说是另一套了
如果要起新项目一点点迁移那就直接用官方支持的vite新建项目而不是用这个半吊子的...
